### ISSUE TITLE
Talaria config is not live loaded

### ISSUE THREAD

**Issue by atlas-comstock** [kelindar/talaria#89]
The config was designed to be live-loaded. However, it is a local variable inside the `main function [here](https://github.com/kelindar/talaria/blob/master/main.go#L47)

**Comment by kelindar** [comment]
Indeed, we should propagate this function instead of retrieving the value once.

**Comment by tardunge** [comment]
+1. Can be handy to have filters and encoders defined dynamically for sinks.
